ui/packages/gallery/src/Gallery.svelte

```diff
@@ -15,6 +15,7 @@
 	export let root_url: null | string = null;
 	export let value: Array<string> | Array<FileData> | null = null;
 	export let style: Styles = { grid: [2], height: "auto" };
+	let styles: string;
```

Comment:
We can remove this i think, we don't need to define the variable upfront when using `$: x = y`. The type should be inferred.